<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReleaseMediaDialog</class>
 <widget class="QDialog" name="ReleaseMediaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="rootLayout">
   <item>
    <widget class="QPushButton" name="showPreviousButton">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/angle-left.svg</normaloff>:/icons/angle-left.svg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="contentStackedWidget">
     <widget class="QWidget" name="imagePage">
      <layout class="QVBoxLayout" name="imagePageLayout">
       <item>
        <widget class="QLabel" name="imageLabel">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="videoPage">
      <layout class="QVBoxLayout" name="videoPageLayout">
       <item>
        <widget class="QWebEngineView" name="videoView">
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loadingPage">
      <layout class="QVBoxLayout" name="loadingPageLayout">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="maximum">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="showNextButton">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/angle-right.svg</normaloff>:/icons/angle-right.svg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
